geest/core/workflows/workflow_base.py

```diff
@@ -637,3 +639,61 @@ def _combine_rasters_to_vrt(self, rasters: list) -> None:
             return False
 
         return vrt_filepath
+
+    def _assign_reclassification_to_polygons(
```

Comment:
We should only add things to the base class if we expect them to be used from multiple concrete classes. Otherwise put this in its own module or in the concrete class